fastify.js

```diff
@@ -280,6 +280,10 @@ function fastify (options) {
     route: function _route (options) {
       // we need the fastify object that we are producing so we apply a lazy loading of the function,
       // otherwise we should bind it after the declaration
+      
+      if (typeof options.method === 'string') {
+            options.method = options.method.toUpperCase();
```

Comment:
Note that this `method` could be an array, so we should support `method: ['get', 'POST']` too